mitiq/pt/__init__.py

```diff
@@ -3,4 +3,8 @@
 # This source code is licensed under the GPL license (v3) found in the
 # LICENSE file in the root directory of this source tree.
 
-from mitiq.pt.pt import execute_with_pauli_twirling
+from mitiq.pt.pt import (
+    execute_with_pauli_twirling,
+    _generate_lookup_table,
+    sample_paulis,
+)
```

Comment:
Probably don't need to expose the "private" `_generate_lookup_table` function at this level, right?